gordo_components/builder/build_model.py

```diff
@@ -70,28 +77,44 @@ def build_model(
     logger.debug(f"Initializing Model with config: {model_config}")
     model = serializer.pipeline_from_definition(model_config)
 
-    # Cross validate
-    logger.debug(f"Starting to do cross validation")
-    start = time.time()
-
-    scores: Dict[str, Any]
-    if hasattr(model, "score"):
-        cv_scores = cross_val_score(model, X, y, cv=TimeSeriesSplit(n_splits=3))
-        scores = {
-            "explained-variance": {
-                "mean": cv_scores.mean(),
-                "std": cv_scores.std(),
-                "max": cv_scores.max(),
-                "min": cv_scores.min(),
-                "raw-scores": cv_scores.tolist(),
+    cv_duration_sec = None
+
+    if cv_mode == "cross_val_only" or cv_mode == "full_build":
```

Comment:
minor, `if cv_mode.lower() in ("cross_val_only", "full_build"):` ever so slightly easier to read; also `.lower()` maybe help the frustrated user. :sweat_smile: